### ISSUE TITLE
A typo in USAGE.md

### ISSUE THREAD

**Issue by FrankWu100** [ptitSeb/gl4es#64]
Hi, 

I found some typo as below:
Path: [USAGE.md](https://github.com/ptitSeb/gl4es/blob/master/USAGE.md)
[L15](https://github.com/ptitSeb/gl4es/blame/master/USAGE.md#L15): Controls ~~**`tje`**~~ **the** version of GLES to use

also, are they duplicated item?
1. [L14](https://github.com/ptitSeb/gl4es/blame/master/USAGE.md#L14): LIBGL_ES
    [L229](https://github.com/ptitSeb/gl4es/blame/master/USAGE.md#L229): LIBGL_ES

2. [L20](https://github.com/ptitSeb/gl4es/blame/master/USAGE.md#L20): LIBGL_GL
    [L235](https://github.com/ptitSeb/gl4es/blame/master/USAGE.md#L235): LIBGL_GL

Thanks.

**Comment by ptitSeb** [comment]
Yes, you are right. It should be fixed with 42a6bd8046852bd781f33da0830741050c67e9cb
